spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -1693,7 +1693,10 @@ properties (with the prefix stripped) when the local `EntityManagerFactory` is c
 
 [[howto-configure-hibernate-naming-strategy]]
 === Configure Hibernate Naming Strategy
-Hibernate defines `Physical` and `Implicit` naming strategies. Spring Boot configures
+Since version 5, Hibernate defines `Physical` and `Implicit` naming strategies which you
+can configure in Spring Boot by providing a fully qualified class name to the properties
+`spring.jpa.hibernate.naming.physical-strategy` and
+`spring.jpa.hibernate.naming.implicit-strategy`. Spring Boot configures
```

Comment:
I find the names `physical- strategy` and `implicit-strategy` quite self explanatory. If you use your IDE, you'll easily get a match for implicit and physical. Ditto if you browe the appendix using your browser. Is there a reason why you need to repeat the name of the properties here?